### PR TITLE
[MUL-316] Unify vendor panel breadcrumbs and head titles

### PR DESCRIPTION
**Pull request by Kriz83**
Change breadcrumbs generation process for vendor account parts, change and fix incorrect or inconsistent titles

### CODE REVIEW

File: src/Component/Core/Api/Resources/api_resources/ProductVariant.xml [hunk 1]
```diff
@@ -43,27 +43,27 @@ We are hiring developers from all over the world. Join us and start your new, ex
 
             <itemOperation name="vendor_inventory_get">
```
Comment (senghe) [inline src/Component/Core/Api/Resources/api_resources/ProductVariant.xml:44]: This file cannot be changed

File: src/Component/Core/Api/Resources/serialization/ProductVariant.xml [hunk 2]
```diff
@@ -12,22 +12,22 @@ We are hiring developers from all over the world. Join us and start your new, ex
 >
     <class name="Sylius\Component\Core\Model\ProductVariant">
```
Comment (senghe) [inline src/Component/Core/Api/Resources/serialization/ProductVariant.xml:13]: This file also shouldn't be changed

File: src/Component/Core/Vendor/Twig/Extension/BreadcrumbExtension.php [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        }
+
+        if (count($pathElements) - 1 === $key) {
+            $activeValue = true;
+            $routeName = null;
+        }
+
+        $resultArray[$element] = [
+            'trans' => $translatable,
+            'active' => $activeValue,
```
Comment (senghe) [inline src/Component/Core/Vendor/Twig/Extension/BreadcrumbExtension.php:96]:
```suggestion
            'translation' => $translatable,
            'activeValue' => $activeValue,
```

File: src/Component/Core/Vendor/Twig/Extension/BreadcrumbExtension.php [hunk 4]
```diff
@@ -0,0 +1,142 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
```
Comment (senghe) [inline src/Component/Core/Vendor/Twig/Extension/BreadcrumbExtension.php:9]: I asked @Kriz83 on a call to use the strategy pattern for this. The _generatedBreadcrumb.html.twig file can stay.

File: templates/Context/Vendor/Settlement/create.html.twig [hunk 5]
```diff
@@ -1,22 +1,14 @@
...
-        <a href="{{ path('sylius_shop_homepage') }}" class="section">{{ 'sylius.ui.home'|trans }}</a>
-        <div class="divider"> / </div>
-        <a href="{{ path('sylius_shop_account_dashboard') }}" class="section">{{ 'sylius.ui.my_account'|trans }}</a>
-        <div class="divider"> / </div>
-        <a href="{{ path('open_marketplace_vendor_virtual_wallet_index') }}" class="section">{{ 'open_marketplace.ui.virtual_wallet'|trans }}</a>
-        <div class="divider"> / </div>
-        <div class="active section">{{ channel.name  ~ ' - ' ~ 'open_marketplace.ui.profit_withdrawal'|trans }}</div>
-    </div>
+{% block breadcrumb_page %}
+    {% include 'Context/Vendor/Common/_generatedBreadcrumb.html.twig' %}
```
Comment (senghe) [inline templates/Context/Vendor/Settlement/create.html.twig:11]:
```suggestion
    {% include 'Context/Vendor/Common/_generatedBreadcrumb.html.twig' with {"context": "settlement-create"} %}
```

**Review by senghe — CHANGES_REQUESTED** [review]

File: src/Component/Core/Vendor/Twig/Extension/BreadcrumbExtension.php (hunk 4 above)
Comment (Kriz83) [reply src/Component/Core/Vendor/Twig/Extension/BreadcrumbExtension.php:9]: it is now

**Review by senghe — APPROVED** [review]

**Review by BartoszWojdalowicz — APPROVED** [review]

**Pull request merged**